Tests/feaLib/data/STAT_test.fea

```diff
@@ -9,10 +9,14 @@ table STAT {
                          name 3 1 1041 "ローマン";
                          };
 
-    DesignAxis opsz 0 { name "Optical Size"; };
-    DesignAxis wdth 1 { name "Width"; };
-    DesignAxis wght 2 { name "Weight"; };
-    DesignAxis ital 3 { name "Italic"; };  # here comment
+    DesignAxis opsz 0 { name "Optical Size";
+    };
```

Comment:
why this?

Reply:
Why what? New line? I just updated now so it's consistent with the rest. In practice people can do whatever they want with the whitespace, but here we expect newlines for each name entry since there can be multiple.